web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -318,18 +312,6 @@ Disadvantages:
 Given that `inplace` is already widely used by the pandas community, we would like to collect feedback about what the
 expected return type should be. Therefore, we will defer a decision on this until a later revision of this PDEP.
 
-## Backward compatibility
-
-Removing the `inplace` keyword is a breaking change, but since the affected behaviour is `inplace=True`, the default
-behaviour when not specifying the keyword (i.e. `inplace=False`) will not change and the keyword itself can first be
-deprecated before it is removed.
-
-Similarly for the `copy` keyword, this can be deprecated before it is removed.
-
-There are some behaviour changes (for example the current `copy=False` returning a shallow copy will no longer be an "
-actual" shallow copy, but protected under Copy-on-Write), but those behaviour changes are covered by the Copy-on-Write
-proposal[^1].
-
```

Comment:
I removed this since its mostly redundant/duplicate information with e.g. the Timeline section.

The deprecation process is not different from any other breaking change, so there shouldn't be a need to have a dedicated section.

Reply:
I would personally keep such a section. Even if trivial, I think it is good to have an explicit "Backward compatibility" section in every PDEP (even if this would just say "no impact" for something that is a new feature, just to be explicit). 
(for example, if we would have added a template for PDEPs, I would have argued that this should be a standard section ;))

https://github.com/lithomas1/pandas/pull/33 is also already shortening this section quite a bit, so reducing the possible redundancy

Reply:
That's fine by me I guess. 
I took a look at some NEPs, and some of them do have a "Backwards Compatibility" section(like https://numpy.org/neps/nep-0018-array-function-protocol.html#backward-compatibility).

I think it would be nice to keep the Timeline/Backcompat section together, though (maybe we can have them as subheadings under a "Migration Process" section).